examples/jwt/src/main.rs

```diff
@@ -150,6 +150,21 @@ impl IntoResponse for AuthError {
     }
 }
 
+#[derive(Debug)]
+struct Keys {
+    encoded: EncodingKey,
+    decoded: DecodingKey<'static>,
```

Comment:
One more nit, I think `encoding` and `decoding` makes more sense for these, since the key itself isn't what's being de- / encoded.

Reply:
Done. Thanks a lot! Now I'm going to implement https://github.com/tokio-rs/axum/pull/308#discussion_r704077062.